project/Dependencies.scala

```diff
@@ -1,120 +1,117 @@
-  // It's possible that after updating all other dependencies in Cromwell we might purge the transitive dependencies on
-  // javax.activation and then be able to upgrade to jaxb-impl 2.3.3 or beyond, but some of those other dependencies
-  // such as googleCloudNioV have already been pinned for Scala Steward so this might not be a trivial undertaking.
-  private val jaxbV = "2.3.2" // scala-steward:off
-  private val kindProjectorV = "0.9.9"
-  private val kittensV = "2.2.1"
+  private val hsqldbV = "2.6.0"
+  private val http4sV = "0.21.7" // scala-steward:off (CROM-6678)
+  private val jacksonV = "2.12.5"
+  private val janinoV = "3.1.6"
+  private val jsr305V = "3.0.2"
+  private val kindProjectorV = "0.9.10"
+  private val kittensV = "2.3.2"
   private val liquibaseSlf4jV = "4.0.0"
-  // Scala Steward wanted to upgrade liquibase-core to 3.10.2 but that version does not find some uniqueness
-  // constraints and models datatypes in ways that are incompatible with our test expectations.
-  // liquibase-core 4.0.0 did not have either of those problems but produced tons of strange warnings at runtime
-  // similar in form to this: https://github.com/liquibase/liquibase/issues/1294
-  // Pinning Liquibase version for the time being.
-  private val liquibaseV = "3.6.3" // scala-steward:off
-  private val logbackV = "1.2.3"
-  private val lz4JavaV = "1.7.1"
-  private val mariadbV = "2.7.2"
-  private val metrics3ScalaV = "4.0.0" // https://github.com/erikvanoosten/metrics-scala/tree/f733e26#download-4x
+  private val liquibaseV = "4.4.0" // 4.4.1+ needs https://github.com/liquibase/liquibase/pull/2001
+  private val logbackV = "1.2.5"
+  private val lz4JavaV = "1.8.0"
+  private val mariadbV = "2.7.4"
+  /*
+  The StatsD reporter for DropWizard's (Code Hale's) Metrics 3.x still works with Metrics 4.x.
+  Still would be great to move to Prometheus / OpenCensus
+   */
+  private val metrics4ScalaV = "4.1.19"
   private val metrics3StatsdV = "4.2.0"
   private val mockFtpServerV = "2.8.0"
-  private val mockserverNettyV = "5.5.4" // scala-steward:off (CROM-6582)
-  private val mouseV = "1.0.0"
-  private val mysqlV = "8.0.22" // scala-steward:off (CROM-6772: DateTime/String conversion and connection string updates required)
-  private val nettyV = "4.1.46.Final"
-  private val owlApiV = "5.1.16" // scala-steward:off (CROM-6677)
+  private val mockserverNettyV = "5.11.2"
+  private val mouseV = "1.0.4"
+  private val mysqlV = "8.0.26"
+  private val nettyV = "4.1.66.Final"
+  private val owlApiV = "5.1.19"
   private val paradiseV = "2.1.1"
   private val pegdownV = "1.6.0"
-  // For org.postgresql:postgresql 42.2.6 - 42.2.14:
-  // java.lang.NoSuchFieldException: m_mesgParts in KeyValueSpec "fail if one of the inserts fails"
-  private val postgresV = "42.2.5" // scala-steward:off
-  private val rdf4jV = "2.4.2"
-  private val refinedV = "0.9.22"
+  private val postgresV = "42.2.23"
+  private val pprintV = "0.6.6"
+  private val rdf4jV = "3.7.1"
```

Comment:
Ugh. Their link to the "release and upgrade notes" (https://rdf4j.org/release-notes#3-0-0) for 3.0.0 doesn't go to a useful place anymore so I don't know what to look for. I also don't know what RDF is or how we might use it.

Reply:
- RDF: https://en.wikipedia.org/wiki/Resource_Description_Framework
- It's one of the specs used by the CWL spec ([example](https://pypi.org/project/cwltool/#modeling-a-cwl-document-as-rdf))
- This repo only uses rdf4j transitively. However, there was a vulnerability detected in an earlier version, and the OWL API dependency (yet another W3C spec) we use was slow to update their dependencies.
- For the curious about these libs in general there are [more external links regarding dependencies here](https://github.com/broadinstitute/cromwell/blob/67/project/Dependencies.scala#L450-L463)